samples/ios-unified/Sample.csproj

```diff
@@ -62,12 +62,13 @@
     <ConsolePause>false</ConsolePause>
     <MtouchArch>ARMv7, ARM64</MtouchArch>
     <CodesignEntitlements>Entitlements.plist</CodesignEntitlements>
-    <MtouchProfiling>true</MtouchProfiling>
+    <MtouchProfiling>false</MtouchProfiling>
```

Comment:
What's this one about?

Reply:
I can't remember exactly. It may have changed as a side-effect of another change I made.